packages/RichTextEditing-Core.package/RichTextEditorModel.class/instance/positionInTextAtIndex..st

```diff
@@ -0,0 +1,14 @@
+document information
+positionInTextAtIndex: anIndex
+
+	| lineCounter currentLine positionInLine |
+
+	lineCounter := 0.
+	currentLine := self displayedLines 
+		detect: 
+			[: each | lineCounter := lineCounter + 1. 
+			each first <= (anIndex + 1) and: [each last >= (anIndex + 1)]] 
+		ifNone: [self displayedLines last].
+	positionInLine := anIndex - currentLine first + 2.
+	
+	^ positionInLine @ lineCounter
```

Comment:
I'm not sure if I like the point here or if it's better to split it into two different methods

Reply:
Good point, I also thought about that. Not quite sure what I prefer though

Reply:
A point is fine imo, it represents the context better than two methods, as the position in the text is here really a position in form of x@y